src/components/DaysPanel/index.tsx

```diff
@@ -0,0 +1,109 @@
+import { IonContent, IonItem, IonLabel, IonList, IonRadioGroup, IonRadio } from '@ionic/react';
+import React, { useState } from 'react';
+import  ActionableModalCard  from '../ModalCards/ActionableModalCard'
+import './style.scss';
+import store from '../../store';
+
+
+
+
+
+const DaysPanel : React.FC = () => {
+    
+    const[onHoldProps,setOnHoldProps] = useState( {showModal: false, days : [''], selectedDay:'' } );

```

Comment:
these 3 are changing independently, so its recommended to use 3 useStates, that increases readability

Reply:
I'd thought that may be in future in case we have more usecases then more states would have needed for this component, so usecase wise I've composed state object